### DISCUSSION TITLE
Large .log file created during simulations

### DISCUSSION THREAD

**Discussion by Qcellaris** [pypr/pysph#392] [Q&A]
I have been using PySPH for a while on one machine and currently I was trying to use it on another machine which has nearly the same hardware as the first machine. Nevertheless, I figured that the simulations on the new machine took significantly more time. I think it has to do with the fact that a .log file is created of several GB while on my old installation it was only about 100 KB. When I checked the .log files it shows lines that begin with DEBUG. In the old .log files I didn't see these lines with DEBUG, I only found lines beginning with INFO. Any idea where this came from? My old machine runs PySPH 1.0b1.dev0 while my new machine runs PySPH 1.0b2.dev0.

**Comment by nauaneed** [comment]
Can you see if you have this somewhere:

```
import logging

logger = logging.getLogger(__name__)
logger.setLevel(logging.DEBUG)
```

You can set the logger to print only up to INFO level logs with:

```
logger.setLevel(logging.INFO)
```

Please try this.

**Reply by Qcellaris** [reply]
It already happens when I run an example from the examples directory (so no code I made myself). Those examples do not import the logging module. I think it is set somewhere in the PySPH code itself. I installed PySPH by downloading the latest version via Git and then I ran python setup.py install. Maybe I should just try an older version of PySPH?

**Reply by Qcellaris** [reply]
I did a complete reinstall of my python environment and now suddenly it works. I don't know what caused the issue, but apparently it had something to do with package versions.

**Reply by Qcellaris** [reply]
Now I recently reinstalled PySPH on the same machine as the first machine, and it also writes these very long logging files by default. What is the default logger.setLevel value in the PySPH code? I can't seem to find it anywhere. I think maybe in one of the version updates of PySPH the default logging level got changed. Is that possible?
